test/generator.hpp

```diff
@@ -88,24 +89,24 @@ namespace eve::test
   template<typename V> auto ramp(V v)
   {
     return  [=]<typename T>(eve::as_<T>, auto&)
-            {
-              std::array<eve::element_type_t<T>, amount<T>()> d;
-              for(std::size_t i = 0;i<amount<T>();++i) d[i] = v+i;
-              return d;
-            };
+    {
```

Comment:
This is uneeded changes

Reply:
this is uneedded comment: clang-format will decide, it's just how my editor indented the text
it's perhaps unneeded but harmless
I made real changes in the file elsewhere

Reply:
That's the issue, your change is 1 line but the main thing I see when reviewing is blocks of tens of lines of change. Please check your local editor setup. I know we'll clans-format but this is not the reason to introduce noises that akes reviews more complex.